03/Product.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+function Product(props) {
+  const { name, price, isCart, isCategory, id, addItem, deleteItem } = props;
+  return (
+    <li>
+      <p>{name}</p>
+      <p>{price}</p>
+      <button
+        onClick={()=>addItem(id, name, price, isCart)}
+        disabled={isCategory ? false : true}
+      >
+        Dodaj
+      </button>
+      <button
+        disabled={isCart ? false : true}
+        onClick={()=> deleteItem(id)}
+      >
```

Comment:
Jeśli masz ochotę to możesz spróbować utworzyć osobny komponent `<Button>`, który będzie wyglądał i zachowywał się w zależności od przekazanych `props`.